code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -120,11 +120,21 @@
 	last_healed = world.time + heal_interval
 
 /obj/effect/alien/resin/special/pylon/core/attack_alien(mob/living/carbon/Xenomorph/M)
+	if(!hardcore && last_attempt + 5 SECONDS > world.time)
```

Comment:
make it only trigger the destruction stuff on harm intent so that builders can still heal it